Kevin yoo/setup context flaskdata (#3110)
hubmapconsortium/portal-ui: CHANGELOG-hmp-122-setup-context-flaskdata.md

> - Context was set up in the App file to begin transition of prop drilling to context.

Reviewer: ```suggestion
- Add a context provider to pass flask data to react components.
```
I think it was a little vague as to what data was being passed by context.

> - Prop drilling to context was completed for Donor page.

Reviewer: ```suggestion
- Pass flask data to Donor page components by context.
```
Small, but I'd like to match the tense of other changelog entries. It's small and not a huge issue if we're at times inconsistent, but worth noting.
